programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -15,15 +15,27 @@
 #error "please port this file to your operating system"
 #endif
 
+extern dns_manager *get_dnsmasq_manager(const char* path);
+
+struct ziti_instance_s {
+    ziti_options opts;
+    ziti_context ztx;
+    LIST_ENTRY(ziti_instance_s) _next;
+};
+
+// temporary list to pass info between parse and run
+static LIST_HEAD(instance_list, ziti_instance_s) instance_init_list;
+
+// map<path -> ziti_instance>
+static model_map instances;
+
 static void on_ziti_event(ziti_context ztx, const ziti_event_t *event);
 
 const char *cfg_types[] = { "ziti-tunneler-client.v1", "intercept.v1", "ziti-tunneler-server.v1", "host.v1", NULL };
-static ziti_options OPTS = {
-        .config_types = cfg_types,
-        .events = ZitiContextEvent|ZitiServiceEvent,
-        .event_cb = on_ziti_event,
-        .refresh_interval = 10, /* default refresh */
-};
+
+static long refresh_interval = 10;
```

Comment:
10 on purpose? shouldn't this be more like 300?

Reply:
this was the default, I am not changing that